docs/administration/security.md

```diff
@@ -1,5 +1,627 @@
+
+::: tip
+
+The supported version of Apache is 2.4 which should be used in a supported version of OS. You can learn more about supported environments [here](https://www.exoplatform.com/terms-conditions/supported-environments.pdf).
+:::
+
+##### Required modules
+
+You need mod_ssl, mod_proxy. They are all standard Apache2 modules, so no installation is required. You just need to enable them with the following command:
+
+```shell
+    sudo a2enmod ssl proxy proxy_http headers
+```
+
+##### Configuring a virtual host for the SSL port
+
+Add this to site configuration (you can override the default ssl site `/etc/apache2/sites-enabled/default-ssl.conf` or create your own site):
+
+```xml
+    <VirtualHost *:80>
+        ServerName proxy1.com
+        Redirect / https://proxy1.com/
+    </VirtualHost>
+
+    <VirtualHost *:443>
+        ServerName proxy1.com
+        ProxyPass / http://exo1.com:8080/
+        ProxyPassReverse / http://exo1.com:8080/
+        ProxyRequests Off
+        ProxyPreserveHost On
+        RequestHeader set "X-Forwarded-Proto" expr=%{REQUEST_SCHEME}
+
+        ProxyPass /cometd ws://exo1.com:8080/cometd max=200 acquire=5000 retry=5 disablereuse=on flushpackets=on
+
+        SSLEngine On
+        SSLCertificateFile /path/to/folder/from/certificate/cert.pem
+        SSLCertificateKeyFile /path/to/folder/from/certificate/cert-key.pem
+    </VirtualHost>
+```
+
+#### Configuring Nginx
+
+Instructions for installing Nginx can be found [here](http://wiki.nginx.org/Install). On Debian and Ubuntu you can install Nginx with the following command:
+
+```shell
+   apt-get install nginx
+```
+
+Configure the server *proxy1.com* at port *443* like this (you can put the configuration in a file like `/etc/nginx/sites-enabled/proxy1.com`):
+
+```nginx
+    server {
+        listen 80;
+        server_name proxy1.com;
+
+        # Redirect all HTTP requests to HTTPS with a 301 Moved Permanently response.
+        return 301 https://$host$request_uri;
+    }
+
+    server {
+        listen 443;
+        server_name proxy1.com;
+        ssl on;
+        ssl_certificate /path/to/file/mycert.pem;
+        ssl_certificate_key /path/to/file/mykey.pem;
+
+        location / {
+            proxy_pass http://exo1.com:8080;
+        }
+        location /cometd/cometd {
+            proxy_pass http://exo1.com:8080;
+            proxy_http_version 1.1;
+            proxy_set_header Upgrade $http_upgrade;
+            proxy_set_header Connection "upgrade";
+        }
+
+    }
+ ```
+
+The configuration here is a simple one and it works. For a more advanced configuration, you may want to read [this blog post](http://blog.exoplatform.com/en/2014/04/17/apache-2-nginx-highly-secure-pfs-ssl-encrypting-reverse-proxy-exo-platform-4-0-web-application).
+
```

Comment:
The link is KO. We should change this part :
```suggestion
The configuration here is a simple one and it works. It can be tuned for advanced usages.

```